setup.py

```diff
@@ -18,8 +18,11 @@
       description='An OpenAI gym environment for electric motor control.',
       packages=setuptools.find_packages(),
       install_requires=requirements,
-      extras_require={'examples': 
-                      ['keras-rl2 @git+https://github.com/wau/keras-rl2.git']
+      extras_require={'examples': [
+                          'keras-rl2 @git+https://github.com/wau/keras-rl2.git',
+                          'stable-baselines3',
+                          'tensorforce==0.5.5',
```

Comment:
Is it necessary to restrict tensorforce to one version?

Can't we require any version or at least >=0.5.5 such that per default the newest version will be installed?

Reply:
we tested different versions of tensorforce on different machines and only 0.5.5 will run our examples without errors or will not require any changes